arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,562 @@
+        }
+      }
+    },
+    "RpSkuTier": {
+      "description": "Represents the SKU name and tier",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "enum": [
+            "S1",
+            "S2",
+            "S4"
+          ],
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsString": false
+          },
+          "description": "Name of the Sku level for the server being provisioned."
+        },
+        "tier": {
+          "type": "string",
+          "enum": [
+            "Developer",
+            "Standard"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": false
+          },
+          "description": "Name of the tier to which the Sku applies."
+        }
+      },
+      "required": [
+        "name",
+        "tier"
+      ]
+    },
+    "RpPropertyBag": {
+      "description": "Property bag",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current state of the provisioning operation"
+        },
+        "serverFullName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Full URI of the Analysis Server"
+        },
+        "asAdministrators": {
+          "$ref": "#/definitions/RpAsAdministrators",
+          "description": "List of users to be made administrators on the server"
+        }
+      }
+    },
+    "RpAsAdministrators": {
+      "description": "Array of administrator user identities",
+      "type": "object",
+      "properties": {
+        "members": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "UPN of the user. Example johnsmith@contoso.com."
+          },
+          "description": "Collection of administrator user identities"
+        }
+      }
+    },
+    "RpProvisionRequest": {
+      "description": "Provision request specification",
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Location of the Analysis server"
+        },
+        "sku": {
+          "$ref": "#/definitions/RpSkuTier",
+          "description": "Sku of the Analysis server"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": { "type": "string" },
+          "description": "Key value pairs of additional properties that can ebe specified"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RpPropertyBag",
+          "description": "Provision request properties"
+        }
+      },
+      "required": [
+        "location", 
+        "sku"
+        ]
+    },
+    "RpProvisionResult": {
+      "description": "Provision result specification",
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
This is a result object. We don't need to enforce readonly. I see other swagger spec do similar thing.